src/models/endpoint/participantEndpoint.js

```diff
@@ -0,0 +1,77 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Shashikant Hirugade <shashikant.hirugade@modusbox.com>
+ --------------
+ ******/
+
+'use strict'
+
+const Logger = require('@mojaloop/central-services-shared').Logger
+const Config = require('../../lib/config')
+const Mustache = require('mustache')
+const request = require('request')
+
+/**
+* @function fetchEndpoints
```

Comment:
Please align this to the function name as per the model in the Central-Ledger, which should just be `GetEndpoint` as follows:
```JSON
/**
 * @function GetEndpoint
 *
 * @async
 * @description This retuns the active endpoint value for a give participantId and type of endpoint
 *
 *
 * @param {integer} participantId - the id of the participant in the database. Example 1
 * @param {string} type - the type of the endpoint. Example 'FSIOP_CALLBACK_URL'
 *
 * @returns {array} - Returns participantEndpoint array containing the details of active endpoint for the participant if successful, or throws an error if failed
 */

const getEndpoint = async (participantId, endpointType) => {}
```